<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="DropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(0, 240, 112);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>581</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>&lt;Strong&gt; HSRW Laser tool</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_subtitle">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>581</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 186, 84);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>&lt;Strong&gt;V 1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>471</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	background-color:rgb(0, 186, 84);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}

QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 85, 127, 255), stop:1 rgba(0, 0, 0, 255));
}</string>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>581</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 186, 84);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>350</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 186, 84);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Created by&lt;/span&gt; Arthur Grilo da Silva Santos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>341</width>
         <height>381</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Images/favpng_electrical-network-printed-circuit-board-electronic-circuit-electronics.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>-60</x>
         <y>0</y>
         <width>331</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 100px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Images/5782258-circuit-png-images-vector-and-psd-files-free-download-on-pngtree-circuits-png-360_360.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>label</zorder>
      <zorder>label_title</zorder>
      <zorder>label_subtitle</zorder>
      <zorder>progressBar</zorder>
      <zorder>label_loading</zorder>
      <zorder>label_credits</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
